### ISSUE TITLE
Add benchmark tests for color parsing

### ISSUE THREAD

**Issue by GPHemsley** [servo/rust-cssparser#319]
Spinning off from #314:

With the complexities involved with parsing colors, there are concerned about regressing the time it takes to parse them.

Per @emilio:
> But it should be rather trivial to add a few benchmarks for color parsing in `src/tests.rs` like we have for `data:` uris etc?

We should look into that.

**Comment by sculpt0r** [comment]
Hi :)
Do you think this issue could be a good candidate to start contribute? If so, may I take this one?

**Comment by emilio** [comment]
Hey, sure thing, go ahead!

**Comment by MunishMummadi** [comment]
Hey @emilio @GPHemsley . I am an outreachy applicant can i work on this issue.

**Comment by sculpt0r** [comment]
It has been a long time...
If I'm not mistaken, there is no benchmarking at all at github workflow, despite `bench` feature marks in the test code.

I'm fully rely on the docs: https://doc.rust-lang.org/unstable-book/library-features/test.html

So, part of this issue solve will be adding some custom workflow?

**Comment by emilio** [comment]
I don't think we necessarily need a custom workflow, just something that can be benchmarked easily locally.

**Comment by emilio** [comment]
Of course the more we can automate after that the better, but that's not a strict requirement imo.
